appveyor.yml

```diff
@@ -4,20 +4,20 @@ configuration:
 - Release
 
 install:
-- ps: Start-FileDownload 'https://download.microsoft.com/download/A/3/8/A38489F3-9777-41DD-83F8-2CBDFAB2520C/packages/DotNetCore.1.0.0-SDK.Preview2-x64.exe'
-- cmd: DotNetCore.1.0.0-SDK.Preview2-x64.exe /quiet
+- ps: Start-FileDownload 'https://download.microsoft.com/download/8/F/9/8F9659B9-E628-4D1A-B6BF-C3004C8C954B/dotnet-1.1.1-sdk-win-x64.exe'
+- cmd: dotnet-1.1.1-sdk-win-x64.exe /quiet
```

Comment:
they refer to the same thing. downloading this then doing `dotnet version` will give you 1.0.1. I don't know why this is - MSs versioning confuses me.